src/WSRequestHandler_SceneItems.cpp

```diff
@@ -2,6 +2,85 @@
 
 #include "WSRequestHandler.h"
 
+struct AddSourceData {
+	obs_source_t *source;
+	obs_sceneitem_t *sceneItem;
+	bool setVisible;
+};
+
+void AddSourceHelper(void *_data, obs_scene_t *scene) {
+	auto *data = reinterpret_cast<AddSourceData*>(_data);
+	data->sceneItem = obs_scene_add(scene, data->source);
+	obs_sceneitem_set_visible(data->sceneItem, data->setVisible);
+}
+
+/**
+* Get a list of all scene items in a scene.
+*
+* @param {String} `sceneName` Name of the scene to get the list of scene items from.
+*
+* @return {Array<Object>} `sceneItems` Array of scene items
+* @return {int} `sceneItems.*.itemId` Unique item id of the source item
+* @return {String} `sceneItems.*.sourceId` ID if the scene item's source. For example `vlc_source` or `image_source`
+* @return {String} `sceneItems.*.sourceName` Name of the scene item's source
+* @return {String} `sceneItems.*.sourceType` Type of the scene item's source. Either `input` or `scene`
```

Comment:
Scene items can either be inputs, scenes or groups.

Reply:
If you implement `sourceKind`, specifying `groups` will probably not be necessary